ts/src/flexible-event/privacy.ts

```diff
@@ -214,3 +214,34 @@ function epsilonToBoundInfoGainAndDp(
     return epsilon
   }
 }
+
+// Returns the effective epsilon needed to satisfy an information gain bound
+// given a number of output states in the q-ary symmetric channel.
+export function epsilonToBoundInfoGainAndDp(
```

Comment:
Do you mind commenting briefly what this algorithm is doing at a high level. There's a lot of bit twiddling going on

Reply:
Done!